src/main/java/com/rpietraszewski/medicalclinic/handler/ResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.rpietraszewski.medicalclinic.handler;
+
+import com.rpietraszewski.medicalclinic.exception.PatientEmailAlreadyExistsException;
+import com.rpietraszewski.medicalclinic.exception.PatientIdCardNoChangeException;
+import com.rpietraszewski.medicalclinic.exception.PatientNotFoundException;
+import com.rpietraszewski.medicalclinic.exception.PatientNullFieldsException;
+import com.rpietraszewski.medicalclinic.model.dto.MessageDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.LocalDate;
+
+@ControllerAdvice
+public class ResponseEntityExceptionHandler extends org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler {
```

Comment:
Jesli dasz nad ta klasa ze to RestControllerAdvice to analogicznie jak w przypadku @RestController jestes w stanie zwaracac nie response entity a po prostu MessageDto -> aby ustawić wtedy status odpowiedni należy nad metodą wpisać
@ResponseStatus(HttpStatus.NOT_FOUND);